app/javascript/components/App.js

```diff
@@ -3,13 +3,31 @@ import Home from './pages/Home'
 import Header from './components/Header'
 import Footer from './components/Footer'
 import Navigation from './components/Navigation'
+import PitchCards from './pages/PitchCards'
 import {
   BrowserRouter as  Router,
   Route,
   Switch
 } from 'react-router-dom'
 
 class App extends React.Component {
+  constructor(props){
+    super(props)
+    this.state = {
+      pitchCards: []
+    }
+  }
+
+  componentDidMount(){
+    this.readPitchCard()
+  }
+
+  readPitchCard = () => {
+    fetch("http://localhost:3000/pitchcards")
```

Comment:
Since there is only a single app handling front and back end, you can just fetch to your endpoint `/pitchcards`